db/migrate/20170517233803_add_figure_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddFigureToPieces < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pieces, :figure, :string
```

Comment:
@punkleia I think you many not use either of these columns actually.  Under STI, the `type` column will be the name of the <i>sub</i>class, not the parent, so it will be `Pawn`, `Rook`, `King`, etc.  The connection to Piece is that those classes inherit from Piece in their own definitions.

Each model will have an `id` value automatically assigned, so we shouldn't need `number` to differentiate between the various different pieces since we can use `id` for that.